tests/foreman/api/test_reporttemplates.py

```diff
@@ -68,473 +68,483 @@ def setup_content(request):
         query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
     )[0]
     ak.add_subscriptions(data={'quantity': 1, 'subscription_id': subscription.id})
-    request.cls.org_setup = org
-    request.cls.ak_setup = ak
+    return ak, org
+
+
+# Tests for ``katello/api/v2/report_templates``.
+
+
+@pytest.mark.tier1
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+def test_positive_CRUDL(name):
+    """Create, Read, Update, Delete, List
+
+    :id: a2a577db-144e-4761-a42e-e83885464786
+
```

Comment:
@ColeHiggins2 forgot to mention that we need to add `:parametrized: yes` doc string token for parametrized tests.(after id).

Reply:
This is a blocking comment, as the field is required for parametrized tests.  As a process note, we should not have to catch this manually in review, testimony or our CI pre-commit checks should be able to catch this.